### PR TITLE
Add dns-profile-randomizer tool

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request adds the dns-profile-randomizer tool, which comes from a private Gitlab repository.

## 💭 Motivation and Context

The assessors have requested that this tool be added to the Kali AMI.

## 🧪 Testing

All pre-commit hooks and molecule tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: terraform/user.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Create the test user
+module "user" {
+  source = "github.com/cisagov/molecule-iam-user-tf-module?ref=bugfix%2Fadd-tags-for-role-assumption"
```
Comment (dav3r) [inline terraform/user.tf]: Usual reminder about going back and changing this at some point after that branch gets merged...

**Review by dav3r — APPROVED** [review]
Strong work, as always! 💪 💼

File: README.md [hunk 2]
```diff
@@ -13,11 +13,16 @@ None.
 
 ## Role Variables ##
 
-None.
+In addition to the variables defined in
+[`defaults/main.yml`](defaults/main.yml) (for which the default values
+should suffice) this role requires the following variables:
+
+* `gitlab_token` - the Gitlab personal access token to use when
```
Comment (mcdonnnj) [inline README.md]: Pedantic, but GitLab is PascalCase.

File: defaults/main.yml [hunk 3]
```diff
@@ -71,5 +71,11 @@ manually_installed_tools:
 aquatone:
   - {dir_name: "aquatone", repo: "https://github.com/michenriksen/aquatone/releases/download/v1.7.0/aquatone_linux_amd64_1.7.0.zip"}
 
+# dns-profile-randomizer, which is installed manually but a little
+# differently than the other packages since it is in a private Gitlab
```
Comment (mcdonnnj) [inline defaults/main.yml]: Pedantry about GitLab again.

File: terraform/variables.tf [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  description = "The Cobalt Strike tarball and license objects inside the staging bucket."
+  default     = ["cobaltstrike.tgz", "cobaltstrike.license"]
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "Tags to apply to all AWS resources created"
+
+  default = {
+    Team        = "NCATS OIS - Development"
```
Comment (mcdonnnj) [inline terraform/variables.tf]: This should be "VM Fusion - Development" as it is in `user.tf`.

**Review by mcdonnnj — APPROVED** [review]
You must be tired from all this lifting you do 💪💪💪

File: terraform/variables.tf (hunk 4 above)
Comment (mcdonnnj) [reply terraform/variables.tf]: Alternatively they should both be "CISA - Development" depending on where we are with naming these days.

**Review by felddy — APPROVED** [review]
🍕 !!!!

File: defaults/main.yml (hunk 3 above)
Comment (jsf9k) [reply defaults/main.yml]: See commit 9c32ba2.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit 9c32ba2.

File: terraform/variables.tf (hunk 4 above)
Comment (jsf9k) [reply terraform/variables.tf]: See commit 08682bb.

**Review by mcdonnnj — APPROVED** [review]
Thanks for dealing with my pedantry 👍

File: terraform/user.tf (hunk 1 above)
Comment (jsf9k) [reply terraform/user.tf]: See commit d1af977.

**Pull request merged**